### ISSUE TITLE
Popover 组件与触发元素绑定问题

### ISSUE THREAD

**Issue by nkcjs** [JosephusPaye/Keen-UI#230]
在自定义 `page` 和 `example` 两个组件内使用 `UiPopover` 失败，具体代码如下：
```vue
<template>
  <page title="Popover">
    <example title="基本用法">
      <a ref="trigger1" class="popover-trigger">Click here for the popover</a>
      <ui-popover class="custom-popover" trigger="trigger1">
          Hey there, some popover content here.
          <p>Add <i>whatever</i> content you want here.</p>
      </ui-popover>
    </example>
  </page>
</template>
```
我不是很熟悉Vue，但是经过排查，发现是下面代码的问题
```vue
computed: {
  triggerEl() {
    return this.$parent.$refs[this.trigger];
  }
}
```
通过修改成如下代码能够修复这个BUG:
```vue
computed: {
  triggerEl() {
    let parent = this.$parent;
    let el = null;

    while (!el && parent) {
      el = parent.$refs[this.trigger];
      if (el) return el;
      parent = parent.$parent;
    }

    return el;
  }
}
```

**Comment by JosephusPaye** [comment]
<details>
<summary>Show/hide translation from Google Translate</summary>
Popover component and trigger element binding problem

UiPopover failed in both custom page and example components, with the following code:

I am not very familiar with Vue, but after investigation, found the following code is the problem

By modifying the following code to fix this BUG:
</details>

Hi @webfig,

Thanks for the issue report, it'll be fixed in the next version.

**Comment by Keyves** [comment]
@JosephusPaye `$parent` is reference of the nearest parent vue component, so i think you can use `$vnode.context`, that's what i do
```js
this.$parent.$refs[this.trigger]
```
replaced
```js
this.$vnode.context.$refs[this.trigger]
```

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
